src/main/resources/resources/logisim/strings/fpga/fpga.properties

```diff
@@ -28,6 +28,15 @@ FpgaMapSpecErr = The constant should be specified decimal or hexadecimal (prefix
 #
 FpgaIoPin = Pin
 FpgaIoPins = Pin %d
+dipSwitchZero = Switch 1 left
+dipSwitchMinusNinety = Switch 1 top
+dipSwitchPlusNinety = Switch 1 bottom
+SevenSegmentZero = Segment a top
+SevenSegmentMinusNinety = Segment a right
+SevenSegmentPlusNinety = Segment a left
+LEDArrayZero = row0 col0 left bottom
```

Comment:
Rename to `LedArrayZero` (and following)? We need to unify string naming as well but nonetheless there's `LedRowScanning` and others already.

Also `dipSwitch...` above -> `DipSwitch...`

Reply:
This I leave to the discussion on proper naming for the .properties file